cmd/azure.go

```diff
@@ -29,6 +29,7 @@ import (
 	"net/url"
 	"strconv"
```

Comment:
BTW `admin` APIs are not registered with gateway and also it doesn't make sense to have admin APIs for gateway apart from `admin restart` and `status` to get uptime. 

All calls require underlying real object layer because .. 

- listing locks, clearing locks
- set/getconfig, changing password.
- healing,